cmd/whitesourceExecuteScan_test.go

```diff
@@ -419,7 +419,7 @@ func TestExecuteScanNPM(t *testing.T) {
 		// test
 		err := executeScan(&config, scan, utilsMock)
 		// assert
-		assert.EqualError(t, err, "failed to scan NPM module 'package.json': the file 'package.json/package.json' must configure a name")
+		assert.EqualError(t, err, "failed to scan NPM module 'package.json': the file '"+filepath.Join("package.json", "package.json")+"' must configure a name")
```

Comment:
```
Join("package.json", "package.json")
```

does not look right, does it?

Reply:
I must admit that I haven't looked at semantics here, since I am also not that familiar with the related code parts. That's what was there beforehand...

Reply:
The underlying issue was passing the full file path to the method triggering the error instead of just the path to the directory of the module. I have fixed it with the latest commit.